CriteoMoPubAdapter/Sources/CriteoMoPubAdapter/CRNativeAdAdapter.m

```diff
@@ -54,6 +57,22 @@ - (NSURL *)defaultActionURL {
   return nil;
 }
 
+- (UIView *)mainMediaView {
+  if (self.productMediaView == nil) {
```

Comment:
Instead of having differently named properties to back this interface implementation, what about just directly implementing these?